File: testprojects/python_main/pipelines/DONOT_OPEN_PYTHON/code/org/donot/openme12/graph/all_type_main_1/RowDistributor_1_1_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from org.donot.openme12.config.ConfigStore import *
from org.donot.openme12.udfs.UDFs import *

def RowDistributor_1_1_1(spark: SparkSession, in0: DataFrame) -> (DataFrame, DataFrame):
    df1 = in0.filter((col("`c- short`") > lit(-1)))
    df2 = in0.filter((col("`c- short`") > lit(5)))

    return df1, df2

File: test/sony/scala/livy/pipelines/minimal/code/src/main/scala/io/prophecy/pipelines/minimal/graph/RowDistributor_1_1_1.scala
package io.prophecy.pipelines.minimal.graph

import io.prophecy.libs._
import io.prophecy.pipelines.minimal.config.ConfigStore._
import io.prophecy.pipelines.minimal.config.Context
import io.prophecy.pipelines.minimal.udfs.UDFs._
import io.prophecy.pipelines.minimal.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object RowDistributor_1_1_1 {

  def apply(context: Context, in: DataFrame): (DataFrame, DataFrame) =
    (in.filter(col("year").like(context.config.c_expr)),
     in.filter(col("rme_size_grp").like("%b%"))
    )

}
